BridgeCareApp/VuejsApp/src/components/ReportData.vue

```diff
@@ -2,18 +2,6 @@
     <v-container class="grey lighten-4">
```

Comment:
Figure out if this is still in use. If not, remove it.

Reply:
Don't want to remove it because in this, I have written code to create a table with search functionality. If we need it at some point I can use this code